Maddy-maddigordon757/Homework-2/temp-converter/js/main.js

```diff
@@ -0,0 +1,39 @@
+/*
+    1. Create a function that accepts a temperature value in Fahrenheit and converts it to Celsius.
+      Note: Equation to convert F to C:
+      C = (F - 32) / 1.8
+      Test your function.*/
+    
+    function tempConvert (tempf) {
```

Comment:
Your functions are well written. I encourage you to think about naming -- the other function is `tempConvertC` so it would help you identify which function is which right away if this one also had a non-generic name, like `tempConvertF`.